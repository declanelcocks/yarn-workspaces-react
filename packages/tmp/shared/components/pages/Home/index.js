import React from 'react'
import Helmet from 'react-helmet'
import { add } from '@declan/common'

import config from '../../../../config'

import Heading from '../../atoms/Heading'
import P from '../../atoms/P'

function HomeRoute() {
  return (
    <div>
      <Helmet>
        <title>Home</title>
      </Helmet>

      <Heading level={2}>{config('welcomeMessage')}</Heading>

      <P>Welcome to our boilerplate!</P>
      <P>Hello from @declan/common: {add(1, 2)}</P>
    </div>
  )
}

export default HomeRoute
