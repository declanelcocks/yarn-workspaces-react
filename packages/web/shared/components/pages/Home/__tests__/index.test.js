import React from 'react'
import { shallow } from 'enzyme'

import Home from '../'

describe('<Home />', () => {
  test('renders', () => {
    const wrapper = shallow(<Home />)
    expect(wrapper).toMatchSnapshot()
  })
})
