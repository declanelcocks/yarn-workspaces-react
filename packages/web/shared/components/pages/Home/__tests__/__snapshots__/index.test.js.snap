// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders 1`] = `
<div>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Home
    </title>
  </HelmetWrapper>
  <Heading
    level={2}
    palette="grayscale"
  >
    Hello world!
  </Heading>
  <P>
    Welcome to our boilerplate!
  </P>
</div>
`;
